Dockerfile.alpine

```diff
@@ -0,0 +1,17 @@
+FROM python:3.12-alpine3.20
+
+RUN apk upgrade --update && apk add --no-cache \
+  bash \
+  git \
+  cargo \
```

Comment:
i didn't spend too much time on it, but out of curiosity why do you have a dependency on rust?

Reply:
Hi, thanks for the PR!

Rust is needed to build some python dependencies from source (specifically the evtx package IIRC) but shouldn't actually be needed for production builds.